### PR TITLE
feat: code quality

### PR DESCRIPTION
**Pull request by dni**

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -44,26 +21,28 @@ async def custom_route_handler(request: Request) -> Response:
...
 
+
 def tpos_start():
+    from lnbits.tasks import create_permanent_unique_task
+
     task = create_permanent_unique_task("ext_tpos", wait_for_paid_invoices)
     scheduled_tasks.append(task)
+
+
+__all__ = [
```
Comment (motorina0) [inline __init__.py:42]: was ist das?

File: __init__.py [hunk 2]
```diff
@@ -44,26 +21,28 @@ async def custom_route_handler(request: Request) -> Response:
...
 def tpos_stop():
     for task in scheduled_tasks:
         try:
             task.cancel()
         except Exception as ex:
             logger.warning(ex)
 
+
 def tpos_start():
+    from lnbits.tasks import create_permanent_unique_task
```
Comment (motorina0) [inline __init__.py:36]: why not move also this import up?

File: models.py [hunk 3]
```diff
@@ -77,18 +76,6 @@ def lnurl(self, req: Request) -> Lnurl:
         )
         return lnurl_encode(url)
 
-    def lnurl_response(self, req: Request) -> LnurlWithdrawResponse:
```
Comment (motorina0) [inline models.py:80]: was this not used?

**Review by motorina0 — APPROVED** [review]
LGTM

see my questions in comments

File: models.py (hunk 3 above)
Comment (dni) [reply models.py:80]: yes, also the linter would have caught it

**Pull request merged**